src/main/java/org/mockito/internal/creation/bytebuddy/MockMethodAdvice.java

```diff
@@ -45,6 +47,13 @@
     public MockMethodAdvice(WeakConcurrentMap<Object, MockMethodInterceptor> interceptors, String identifier) {
         this.interceptors = interceptors;
         this.identifier = identifier;
+        try {
+            Field field = WeakConcurrentMap.class.getDeclaredField("target");
```

Comment:
Can you add a code comment explaining why we need this? Please make it thorough, it would be great to share understanding! :)

Reply:
Added a comment.